packages/edit-post/src/components/header/header-toolbar/style.scss

```diff
@@ -39,11 +39,91 @@
 
 	// The Toolbar component adds different styles to buttons, so we reset them
 	// here to the original button styles
-	.edit-post-header-toolbar__left > .components-button.has-icon,
-	.edit-post-header-toolbar__left > .components-dropdown > .components-button.has-icon {
+	.edit-post-header-document-toolbar__left > .components-button.has-icon:first-child {
+		margin-right: 10px;
+	}
+
+	.edit-post-header-document-toolbar__left > .components-button.has-icon,
+	.edit-post-header-document-toolbar__left > .components-dropdown > .components-button.has-icon {
+		height: 32px;
```

Comment:
32px can be $grid-unit-40, 4px can be $grid-unit-05.

Reply:
Yes, for this one I specifically tweaked the px values to match perfectly what's in trunk. Using the grid unit values is 1px off when comparing. Will update using the grid units and check that out.